src/services/dashboards/widgets/aws-data-transfer-cost-trend/AWSDataTransferCostTrendWidget.vue

```diff
@@ -288,7 +288,7 @@ defineExpose<WidgetExpose<Data>>({
         flex-direction: column;
         height: 100%;
         .chart-wrapper {
-            height: 10.75rem;
+            height: 185px;
```

Comment:
question: Is there a reason you used `px` instead of `rem`?

Reply:
yes because chart height must be fixed, not updated by root font size.